extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php

```diff
@@ -56,6 +56,19 @@ public function onEditPageMakeGlobalVariablesScript( array &$aVars ) {
 	}
 
 	/**
+	 * Decide to display Infobox builder in VE
+	 * @param array $aVars
+	 * @return bool
+	 */
+	public function onMakeGlobalVariablesScript( array &$aVars ) {
+		global $wgEnablePortableInfoboxBuilderInVE;
+
+		$aVars['wgEnablePortableInfoboxBuilderInVE'] = $wgEnablePortableInfoboxBuilderInVE;
```

Comment:
is this global add to all pages? shouldn't we limit it only the the ones which can have VE turned on them?